src/framework/execution/logger.ts

```diff
@@ -12,7 +12,7 @@ const bunyanFormat = require('bunyan-format');
 
 interface CreateIntegrationLoggerInput {
   name: string;
-  invocationConfig: IntegrationInvocationConfig;
+  invocationConfig?: IntegrationInvocationConfig;
```

Comment:
Nothing worth changing right now, but I think it may be good to provide a reference to only the `instanceConfigFields` instead of the `invocationConfig`, if the only thing in the config used is the `instanceConfigFields`. This can communicate better what is necessary for the function, and makes tests simpler so that a complete config doesn't have to be built wherever this function is used.

Reply:
Yeah, that would be a good improvement.